README.md

```diff
@@ -1,288 +1,79 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
-
-<!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
```

Comment:
- [ ] As requested by the previous code reviewers All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo. This is mandatory for al the projects onwards otherwise they will be invalidated.

| Expected | Yours |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219669304-8375dfad-2850-49c5-a7d5-fe0231731e11.png) | ![image](https://user-images.githubusercontent.com/58642949/219669588-671a90b8-e8d1-40ac-b9cf-146496b4d022.png) |

Reply:
Hello @franclobo, I really appreciate the review you have done on my project. I want to clarify something.
Do I have to bring the full table of content and write under every main header in the README file even for those that may not be applicable to my project? or do I have to bring the table of content and optionally leave out their description in the README file